deploy/openshift/operator.yaml

```diff
@@ -47,6 +47,15 @@ spec:
           limits:
             memory: "4Gi"
             cpu: "1"
+        livenessProbe:
+          tcpSocket:
+            port: 8383
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          timeoutSeconds: 3
+        readinessProbe:
+          tcpSocket:
+            port: 8383
```

Comment:
since you're at it and you're targeting the same port, I would define the same option as in the liveness probe

```
          initialDelaySeconds: 5
          periodSeconds: 10
          timeoutSeconds: 3
```

Reply:
done: https://github.com/app-sre/deployment-validation-operator/pull/149/commits/0c527a7487dea37bbe9275242a3ab10a1aafa44a

Reply:
really done: https://github.com/app-sre/deployment-validation-operator/pull/149/commits/2e33b8feb03f379d739783adab0e4487c44fb32d